packages/transaction-controller/src/gas-flows/OptimismLayer1GasFeeFlow.ts

```diff
@@ -0,0 +1,84 @@
+import { Contract } from '@ethersproject/contracts';
+import { Web3Provider, type ExternalProvider } from '@ethersproject/providers';
+import { createModuleLogger, type Hex } from '@metamask/utils';
+
+import { CHAIN_IDS } from '../constants';
+import { projectLogger } from '../logger';
+import type {
+  Layer1GasFeeFlow,
+  Layer1GasFeeFlowRequest,
+  Layer1GasFeeFlowResponse,
+  TransactionMeta,
+} from '../types';
+import { buildUnserializedTransaction } from '../utils/layer1-gas-fee-flow';
+
+// This gas flow to be used for the following OP stack chains
+const OPTIMISM_STACK_CHAIN_IDS: Hex[] = [
```

Comment:
Not needed now, but it just occurred to me that our gas API already has a list of Optimism chains, and also an endpoint to return chain IDs, so it might be a quick win in a future ticket to retrieve these chain IDs dynamically from there.

https://github.com/consensys-vertical-apps/va-mmcx-gas-api/blob/main/src/constants.ts#L98